CI/bench/multimodal/custom_user_dir/multimodal_constraints.yaml

```diff
@@ -11,3 +11,4 @@ g4_12x:
   MAX_MACHINE_NUM: 200  
   BLOCK_DEVICE_VOLUME: 300  
   RESERVED_MEMORY_SIZE: 8000
+  TIME_LIMIT: 3600
```

Comment:
Just a note that the `TIME_LIMIT` will be passed to predictor.fit(time_limt=TIME_LIMIT), which terminates the training depending on which comes first, time limit or max_epochs